packages/components/color-picker/src/props.ts

```diff
@@ -0,0 +1,32 @@
+import { buildProps, isValidComponentSize } from '@element-plus/utils'
+import type { PropType } from 'vue'
+import type { ComponentSize } from '@element-plus/constants'
+
+export const colorPickerProps = buildProps({
```

Comment:
please merge this file to `index.ts`

Reply:
acrroding to other component,i create a ts file with same name of .vue component file for props definition